pkg/bgpv1/api/conversions.go

```diff
@@ -261,3 +261,180 @@ func ToAgentRoutes(ms []*models.BgpRoute) ([]*types.Route, error) {
 
 	return ret, nil
 }
+
+func ToAPIRoutePolicies(policies []*types.RoutePolicy, routerASN int64) []*models.BgpRoutePolicy {
+	var ret []*models.BgpRoutePolicy
```

Comment:
Might make sense to pre-alloc the slices here and below if expecting many entries??